doc/controls/TabBarAndTabBarItem.md

```diff
@@ -47,14 +47,16 @@ xmlns:utu="using:Uno.Toolkit.UI"
-| `BadgeVisibility`  | `Visibility`  | Gets or sets the badge visibility of the `TabBarItem`. (Currently only supported by the Material Theme Toolkit Library with `BottomTabBarItemStyle` and `VerticalTabBarItemStyle`)                                                                                                             |
 | `Command`          | `ICommand`    | Gets or sets the command to invoke when the `TabBarItem` is pressed.                                                                                                                                                                                                                           |
 | `CommandParameter` | `object`      | Gets or sets the parameter to pass to the `Command` property.                                                                                                                                                                                                                                  |
 | `Flyout`           | `double`      | Gets or sets the flyout associated with this `TabBarItem`.                                                                                                                                                                                                                                     |
 | `Icon`             | `IconElement` | Gets or sets the icon of the `TabBarItem`.                                                                                                                                                                                                                                                     |
+| `InfoBadge`        | `Control`    | Gets or sets the `InfoBadge` control placed on the top right corner of the `TabBarItem`.                                                                                                                                                                                                                          |
 | `IsSelectable`     | `bool`        | Gets or sets whether the `TabBarItem` can be selected.                                                                                                                                                                                                                                         |
 
+> [!NOTE]
+> The properties `BadgeVisibility` and `BadgeValue` have been deprecated. They were previously used to control the badge visibility and value on the `TabBarItem`. If you need similar functionality, consider using the `InfoBadge` property. The deprecated properties were only supported by the Material Theme Toolkit Library with `BottomTabBarItemStyle` and `VerticalTabBarItemStyle`.
```

Comment:
@kazo0, @Xiaoy312, @MartinZikmund  what do think?
I though maybe we could keep the table as is but with an `(Obsolete)` after the name of the property in the first column in addition to this note. So we don't lose the type and description info as needed (since we are not doing a breaking change here).

Reply:
I'm ok with this description, but your improvement is useful, let's add the Obsolete mention, or an asterisk for example and then start this text with asterisk to show it explains what the asterisk means